calico/_includes/release-notes/v3.23.0-release-notes.md

```diff
@@ -0,0 +1,145 @@
+09 May 2022
+
+#### IPv6 VXLAN support
+
+Calico now supports VXLAN encapsulation for IPv6 networks.
+
+Pull Requests:
+
+ - Add IPv6 VXLAN support [calico #5988](https://github.com/projectcalico/calico/pull/5988) (@coutinhop)
+ - Add IPv6 VXLAN route calculation support to Felix [calico #5919](https://github.com/projectcalico/calico/pull/5919) (@coutinhop)
+ - Allocate IPv6 VXLAN tunnel addresses [calico #5893](https://github.com/projectcalico/calico/pull/5893) (@coutinhop)
+
+#### VPP dataplane beta
+
+The Calico VPP dataplane integration is now in beta!
```

Comment:
"The Calico VPP dataplane integration has reached beta status! Over the last few releases, the VPP team has made great strides in increasing stability, performance and feature compatibility."